lib/widgets/cards/notification_card.dart

```diff
@@ -0,0 +1,191 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import 'package:fokus/model/currency_type.dart';
+import 'package:fokus/services/app_locales.dart';
+import 'package:fokus/utils/app_paths.dart';
+import 'package:fokus/utils/icon_sets.dart';
+import 'package:fokus/utils/theme_config.dart';
+import 'package:fokus/widgets/chips/attribute_chip.dart';
+import 'package:fokus/widgets/cards/item_card.dart';
+
+import 'package:intl/intl.dart';
+
+enum NotificationType{
+  caregiver_receivedAward,
+  caregiver_finishedTaskUngraded,
+  caregiver_finishedTaskGraded,
+  caregiver_unfinishedPlan,
+  child_taskGraded,
+  child_receivedBadge
+}
+
+extension NotificationTypeExtension on NotificationType {
+  static const String _pageKey = 'page.notifications.content';
+
+  String title(BuildContext context, String childName) => {
+    NotificationType.caregiver_receivedAward: childName + " " + AppLocales.of(context).translate("$_pageKey.caregiver.receivedAward"),
+    NotificationType.caregiver_finishedTaskUngraded: childName + " " + AppLocales.of(context).translate("$_pageKey.caregiver.finishedTask"),
+    NotificationType.caregiver_finishedTaskGraded: childName + " " + AppLocales.of(context).translate("$_pageKey.caregiver.finishedTask"),
+    NotificationType.caregiver_unfinishedPlan: childName + " " + AppLocales.of(context).translate("$_pageKey.caregiver.unfinishedPlan"),
+    NotificationType.child_taskGraded: AppLocales.of(context).translate("$_pageKey.child.taskGraded"),
+    NotificationType.child_receivedBadge: AppLocales.of(context).translate("$_pageKey.child.receivedBadge")
+  }[this];
+
+  Icon get icon => Icon(
+    const {
+      NotificationType.caregiver_receivedAward : Icons.star,
+      NotificationType.caregiver_finishedTaskUngraded : Icons.view_headline,
+      NotificationType.caregiver_finishedTaskGraded : Icons.view_headline,
+      NotificationType.caregiver_unfinishedPlan : Icons.cancel,
+      NotificationType.child_taskGraded : Icons.view_headline,
```

Comment:
Instead of lines _view_headline_ I suggest using maybe _assignment_turned_in_:
![obraz](https://user-images.githubusercontent.com/36575365/90693941-040af000-e278-11ea-86a1-5d6636067d2c.png)
And for unfinished plan we should use something different. cancel look to much like an action user can make by tapping on it. Maybe _assignment_late_:
![obraz](https://user-images.githubusercontent.com/36575365/90694016-2dc41700-e278-11ea-877f-315ee6ccf761.png)